.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: '6.0.0'
+    hooks:
+      - id: flake8
+        # Required to make flake8 read fdrom pyproject.toml for now :(
+        additional_dependencies: ["flake8-pyproject"]
+  - repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+      - id: black
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.991
+    hooks:
+      - id: mypy
+        language_version: "3.10"
+        exclude: "bin/"
```

Comment:
Why we need to exclude this directory? All python files should be checked, no?

Reply:
Yes, although, I think those files will be ignored regardless because they aren't named with proper extensions. It has no way of discovering them.

Reply:
Hum well, if it's not used I would not add it (I fear to have copy/paste of this value everywhere, even when repo doesn't have `bin/`)
Out of scope: These scripts should be added as entrypoint to remove this weird workflow

Reply:
Oh no, for sure. I'll remove it. Yes, that would be so much nicer. I was wondering why that wasn't the case already. I just a assumed it was intentional.